### PR TITLE
Task/tec00/gerald devhttps://github.com/Luminescence-inc/Project-Delta-Frontend/pull/109

### PR DESCRIPTION
**Pull request by geraldkachi**
fix conflict

### CODE REVIEW

File: src/components/MultiSearch.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export default function MultiSearch({
 
   return (
     <FlexColStart className="w-full gap-2 relative pb-[20px]">
-      <label className="text-[14px] font-normal font-inter leading-[19px]">
+      <label className="text-sm font-normal font-inter leading-[19px] text-[#0E2D52]">
```
Comment (Benrobo) [inline src/components/MultiSearch.tsx:100]: Please, colors shouldn't be hardcoded. based on the color type, check the `tailwind.config.ts` file and look for similar color name and add that there then reference the color using tailwind classes

File: src/app/page.tsx [hunk 2]
```diff
@@ -23,10 +23,10 @@ const Home = () => {
         <HeaderSearchComp />
         <br />
         <FlexColStart className="gap-[10px]">
-          <h1 className="w-full text-[25px] font-bold leading-[38px] tracking-normal text-left text-blue-200 font-pp">
+          <h1 className="w-full text-[25px] font-bold leading-[38px] tracking-normal text-left text-[##0E2D52] font-pp">
```
Comment (Benrobo) [inline src/app/page.tsx:26]: Same with this

**Review by Benrobo — CHANGES_REQUESTED** [review]
Good job @geraldkachi , but please go through the reviews and make necessary adjustments.

Also please update your pull requests title to easier readability. It should reference the ticket you're working on jira with a short description: `task/TEC-[number]/<short-description>`

File: src/components/ui/button.tsx [hunk 3]
```diff
@@ -84,9 +84,9 @@ const Button: React.FC<ButtonProps> = ({
         ) : (
           // @ts-expect-error
           <Link href={href} relative="path" className={classNames} {...props}>
-            {leftIcon && leftIcon}
-            {children}
-            {rightIcon && rightIcon}
+            <span>{leftIcon && leftIcon}</span>
```
Comment (Benrobo) [inline src/components/ui/button.tsx:87]: Same here

File: src/components/ui/button.tsx [hunk 4]
```diff
@@ -84,9 +84,9 @@ const Button: React.FC<ButtonProps> = ({
...
+            <span>{children}</span>
+            <span>{rightIcon && rightIcon}</span>
```
Comment (Benrobo) [inline src/components/ui/button.tsx:89]: same here

File: src/components/ui/button.tsx [hunk 5]
```diff
@@ -122,8 +122,8 @@ const Button: React.FC<ButtonProps> = ({
           isLoading ? "opacity-0" : "opacity-1"
         )}
       >
-        {leftIcon}
-        {children}
+        <span>{leftIcon}</span>
+        <span>{children}</span>
```
Comment (Benrobo) [inline src/components/ui/button.tsx:126]: Same here

Comment (geraldkachi) [reply src/components/ui/button.tsx:126]: okay removed

File: src/components/ui/button.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (geraldkachi) [reply src/components/ui/button.tsx:87]: okay boss

Comment (geraldkachi) [reply src/components/ui/button.tsx:87]: okay

File: src/components/MultiSearch.tsx (hunk 1 above)
Comment (geraldkachi) [reply src/components/MultiSearch.tsx:100]: hmm okay corrected accordingly

File: src/app/page.tsx (hunk 2 above)
Comment (geraldkachi) [reply src/app/page.tsx:26]: okay boss

File: src/components/ui/button.tsx (hunk 3 above)
Comment (geraldkachi) [reply src/components/ui/button.tsx:87]: the span is an inline tag and it works for the alignment of the icon that's why i put it there.

it slike have three different items in one tag. with the inline it contains it

**Review by osxar — APPROVED** [review]

**Review by osxar — APPROVED** [review]
Code Review Changes has been implemented

**Pull request merged**